os/net/blemgr/bledev.c

```diff
@@ -92,7 +93,7 @@ int bledev_handle(struct bledev *dev, lwnl_req cmd, void *data, uint32_t data_le
 		if (t_server == NULL) {
 			t_server = bledrv_server_get_null_config();
 		}
-		g_scan_queue = (trble_scan_queue *)param.param[1];
+		g_scan_queue = (trble_queue *)param.param[1];
```

Comment:
you should null check data  first. Since you develop middle layer, you should follow that coding style to prevent error in future.